docs/xls-66d-lending-protocol/reference/transactions/loanbrokercoverwithdraw.md

```diff
@@ -0,0 +1,66 @@
+---
+seo:
+    description: Withdraws first-loss capital from a `LoanBroker` ledger entry.
+labels:
+  - Transactions
+  - Lending Protocol
+---
+# LoanBrokerCoverWithdraw
+[[Source]](https://github.com/XRPLF/rippled/blob/ximinez/lending-XLS-66/src/xrpld/app/tx/detail/LoanBrokerCoverWithdraw.cpp "Source")
+
+Withdraws first-loss capital from a `LoanBroker` ledger entry.
+
+Only the owner of the associated `LoanBroker` entry can initiate this transaction.
+
+_(Requires the [Lending Protocol amendment][] {% not-enabled /%})_
+
+
+## Example {% $frontmatter.seo.title %} JSON
+
+```json
+{
+  "TransactionType": "LoanBrokerCoverWithdraw",
+  "Account": "rEXAMPLE9AbCdEfGhIjKlMnOpQrStUvWxYz",
+  "Fee": "12",
+  "Flags": 0,
+  "LastLedgerSequence": 7108682,
+  "Sequence": 8,
+  "LoanBrokerID": "E123F4567890ABCDE123F4567890ABCDEF1234567890ABCDEF1234567890ABCD",
+  "Amount": {
+    "currency": "USD",
+    "issuer": "rIssuer1234567890abcdef1234567890abcdef",
+    "value": "1000"
+  }
+}
+```
+
+
+## {% $frontmatter.seo.title %} Fields
+
+In addition to the [common fields][], {% code-page-name /%} transactions use the following fields:
+
+| Field Name     | JSON Type | Internal Type | Required? | Description |
+|:-------------- |:----------|:-------------|:----------|:------------|
+| `LoanBrokerID` | String    | Hash256      | Yes       | The ID of the `LoanBroker` ledger entry to withdraw from. |
+| `Amount`       | Object    | STAmount     | Yes       | The amount of first-loss capital to withdraw. |
```

Comment:
I think we decided to just call this "Amount" (for the internal type): https://xrpl.org/docs/references/protocol/binary-format#amount-fields